gravitee-apim-console-webui/src/management/api/endpoints-v4/endpoint-group/api-endpoint-group.component.spec.ts

```diff
@@ -235,8 +264,79 @@ describe('ApiEndpointGroupComponent', () => {
           });
         });
       });
-      it('THEN should show the configuration tab', async () => {
+      it('THEN should show the configuration tab and health-check tab', async () => {
         expect(await componentHarness.configurationTabIsVisible()).toEqual(true);
+        expect(await componentHarness.healthCheckTabIsVisible()).toEqual(true);
+      });
+    });
+
+    // Health-check Tab
+    describe('WHEN the Health-check tab is selected', () => {
+      beforeEach(async () => {
+        await componentHarness.clickHealthCheckTab();
+        expectApiSchemaGetRequests(api, fixture, httpTestingController);
+        expectHealthCheckSchemaGet(fixture, httpTestingController);
+      });
+
+      it('THEN the health-check configuration schema should be disabled', async () => {
+        expect(await componentHarness.isHealthCheckConfigurationInputDisabled('dummy')).toBeTruthy();
+      });
+
+      describe('AND WHEN the health-check configuration schema is modified with a valid value', () => {
+        beforeEach(async () => {
+          await componentHarness.toggleEnableHealthCheckInput();
+          await componentHarness.writeToHealthCheckConfigurationValueInput('dummy', VALID_HEALTH_CHECK_VALUE);
+        });
+        it('THEN the health-check enable switch should be toggled', async () => {
+          expect(await componentHarness.isHealthCheckConfigurationInputDisabled('dummy')).toBeFalsy();
+        });
+        it('THEN the health-check value field should be updated with the expected value', async () => {
+          expect(await componentHarness.readHealthCheckConfigurationValueInput('dummy')).toEqual(VALID_HEALTH_CHECK_VALUE);
+        });
+
+        it('THEN the save button should be valid', async () => {
+          expect(await componentHarness.isGeneralTabSaveButtonInvalid()).toBeFalsy();
+        });
```

Comment:
I agree I didn't find them clear and I encountered some problem writing those, but I thought it will be consistent. I will write one test though for health-check